pretty/src/main/java/org/aya/pretty/doc/Doc.java

```diff
@@ -242,6 +242,18 @@ record PageWidth(@NotNull IntFunction<Doc> docBuilder) implements Doc {
   static @NotNull Doc braced(Doc doc) {
     return wrap("{", "}", doc);
   }
+
+  /**
+   * Either `{ defaultDoc }` or `{\nflatDoc\n}`
+   * TODO: There should be a better impl for this! For example, a new Doc element class
+   */
+  static @NotNull Doc flatAltBracedBlock(Doc defaultDoc, Doc flatDoc) {
+    return flatAlt(
+      stickySep(Doc.plain("{"), defaultDoc, Doc.plain("}")),
```

Comment:
Use `Doc.symbol("{")`

Reply:
If you believe in some backend there will be special treatment for the symbol. For example, in LaTeX backend, `{` should be escaped as `\{`, and this is a special treatment, and hence you should use `symbol("{")`. In general, if you _feel_ like something is not some text, then probably symbol.